tsl/test/expected/move.out

```
@@ -0,0 +1,469 @@
+(0 rows)
+
+COMMIT;
+-- Show chunk indexes
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_1_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_2_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+\set ON_ERROR_STOP 0
+-- cannot move a chunk with no reorder index on first call 
+SELECT move_chunk(chunk=>'_timescaledb_internal._hyper_1_2_chunk', destination_tablespace=>'tablespace1', index_destination_tablespace=>'tablespace1');
+WARNING:  Timescale License expired
+ERROR:  there is no previously clustered index for table "_hyper_1_2_chunk"
+-- cannot move a chunk without a destination tablespace set
+SELECT move_chunk(chunk=>'_timescaledb_internal._hyper_1_2_chunk', index_destination_tablespace=>'tablespace1', reorder_index=>'_timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx');
+ERROR:  function move_chunk(chunk => unknown, index_destination_tablespace => unknown, reorder_index => unknown) does not exist at character 8
```

this is not supposed to error like that is it?

---

No, this is fine. It should fail in some way if it's not set, because it's a required argument, i think this is the way it will fail?

---

eh, nvm, set it to null explicitly so it would trigger failure.